bpf/lib/icmp6.h

```diff
@@ -428,6 +452,10 @@ icmp6_host_handle(struct __ctx_buff *ctx __maybe_unused)
 {
 	__u8 type __maybe_unused;
 
+	type = icmp6_load_type(ctx, ETH_HLEN);
+	if (type == ICMP6_NS_MSG_TYPE)
+		return icmp6_handle_ns(ctx, ETH_HLEN, METRIC_INGRESS);
```

Comment:
In this case, we will bypass the host firewall, but that's okay since that's what we do for NS messages anyway. However, could you switch the comment `CTX_ACT_OK` in the table below to `icmp6_handle_ns` for `ICMPv6-neighbor-solicit`?